lib/theme.json

```diff
@@ -450,6 +453,9 @@
 					"style": true,
 					"width": true
 				}
+			},
+			"core/image": {
+				"behaviorsUIEnabled": true
```

Comment:
I don't think this is consistent with the rest of the settings. None of them contain the `UIEnabled` ending.

Reply:
Welcome to the _"naming things in theme.json survivors group"_! 😄 

I take it we want to offer control over how a block "behaves", so "behavior" seems _prima facie_ like an appropriate label, assuming that future uses will fall neatly under the semantic category. 

I was wondering, which behaviors does `behaviorsUIEnabled` encompass? Is it a general flag that will be used in the future for all interactivity-related behaviors? 

So too, will there be further theme-level opt-in features that would dictate how a lightbox will display or function, e.g., animations, video support etc? 

In relation to the schema, the majority of theme.json labels describe specific, visual properties of a block, e.g., `position[ "fixed"]`, `color ["background"]` and so on. This gives us fine-grained control over which features a theme supports, and can be extended to include further properties. For this reason, I would be inclined to take inspiration from the existing block supports nomenclature.

For example, here we're disabled border radius controls for the image block, but explicitly enabling lightbox behavior.

```json
{
	"settings": {
		"blocks": {
			"core/image": {
				"border": {
					"radius": false
				},
				"behaviors": {
					"lightbox": true
				}
			},
		}
	}
}
```

To me, it makes sense to explicitly opt into a single behavior rather than blanket flag so that, as a theme developer, I'm able to fine-tune the controls available in the UI.

Reply:
That's my point. It would be weird to have to write this:

```json
{
  "settings": {
    "blocks": {
      "core/image": {
        "border": {
          "radius": false
        },
        "behaviorsUIEnabled": {
          "lightbox": true
        }
      }
    }
  }
}
```

Reply:
Sorry for not replying earlier; I agree and have followed your thinking on this, Luis & Ramon. Thanks for the comments!